articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Branding
+description: How to brand Auth0 items to reflect your brand and desired user experience
+toc: true
+topics:
+    - b2c
+    - ciam
+    - branding
+    - universal-login
+    - login-pages
+    - change-password-pages
+    - custom-domains
+    - error-pages
+contentType: concept
+useCase:
+    - user-logout
+---
+# Branding
+
+Auth0 can be customized with a look and feel that aligns with your organization's brand requirements and user expectations. The consistency in appearance improves your customer's experience, giving them peace of mind that the pages they're using can be trusted and are secure.
+
+Some of the items that you can brand include:
+
+* Login, Change Password, and Error pages
+* Custom domains
+* Email templates
+
+Auth0 also supports [internationalization (I18N)](/i18n) and localization (L10N), both of which are important if you work with an international clientele.
```

Comment:
Is the preference not to make reference to the Lock UI Widget any more?

Reply:
Good question.  It is all we have for localization right now...  So anyone who needs it and doesn't want to write their own will be stuck with Universal Login (Classic) which is Lock :)

Reply:
Hmmm. It seems that most of the references to the Lock widget from the original B2C Welcome have been removed from the Branding section. I'd like to get a better understanding regarding that, as I feel that removing the references to the Lock widget means we're losing valuable guidance :-(

Reply:
I'm going to resolve this conversation and we should take it up in another, more general discussion. It affects more than this line in the markdown file.